### DISCUSSION TITLE
Include official plugins in the licence checker

### DISCUSSION THREAD

**Discussion by paviro** [pretix/pretix#2055] [Ideas] [answered]
I think it would be nice if plugin hosted by the pretix organisation would not raise a warning in the check result. Any reasons it doesn't know the licence of those plugins?

<img width="990" alt="sdfsdsdf" src="https://user-images.githubusercontent.com/992826/116926404-9c0c0680-ac5a-11eb-8d32-be37a9110037.png">

**Comment by raphaelm** [comment, marked as answer]
Thanks! Can you tell is which versions of these plugins you have exactly? I think we pushed the release to fix the meta data of passbook, but we might have missed the font pack.

**Reply by paviro** [reply]
pretix-fontpack-free 1.6.0 and now also pretix_passbook-1.7.1
passbook was indeed outdated my bad! But after updating and rechecking the licence it still looks the same.

**Reply by fabm3n** [reply]
Same issue with pretix-pages-1.4.1

**Reply by raphaelm** [reply]
@paviro 
Can you still reproduce this with current fontpack-free and passbook versions? pages didn't have a release yet.

**Reply by paviro** [reply]
Nope seems all good 👍 Thanks!
